build/hellobuild.md

```diff
@@ -0,0 +1,109 @@
+
+* Line 6 - This second RUN instruction requires that we’ve installed pip in the layer before. After applying the previous directive, we can use the pip command to install the flask web framework. This is the framework we’ve used to write our basic “Hello World” application from above, so to run it in Docker, we’ll need to make sure it’s installed.
+
+* Line 9 - This COPY instruction copies our `hello.py` file from the build’s context local directory into the root directory of our image. After this executes, we’ll end up with a file called `/hello.py` inside the image, with all the content of our local copy. 
+See also the [COPY instruction](https://docs.docker.com/engine/reference/builder/#copy) page in the Dockerfile reference.
+
+* Line 12 - This ENV instruction sets a linux environment variable we’ll need later. This is a flask-specific variable, that configures the command later used to run our `hello.py` application. Without this, flask wouldn’t know where to find our application to be able to run it. See also the [ENV instruction](https://docs.docker.com/engine/reference/builder/#env) page in the Dockerfile reference.
+
+* Line 13 - This EXPOSE instruction marks that our final image has a service listening on port 8000. This isn’t required, but it is a good practice, as users and tools can use this to understand what your image does. [EXPOSE instruction](https://docs.docker.com/engine/reference/builder/#expose) page in the Dockerfile reference.
+
+* Line14 - This CMD instruction sets the command that is run when the user starts a container based on this image. In this case we’ll start the flask development server listening on all hosts on port 8000. [CMD instruction](https://docs.docker.com/engine/reference/builder/#cmd) page in the Dockerfile reference.
+
+## Test the example
+
+Go ahead and try this example in your local Docker installation or you can use Docker Play that provides you with a temporary Docker instance on the cloud. 
+
+To test this example:
+1. Create a file hello.py with the content of sample A.
+2. Create a file named Dockerfile without an extension with the contents of sample B.
+3. From your Docker instance build it with `docker build -t test:latest .`
+
+    Breaking the docker build command above down:
+
+    `-t` - option and specifying its required paramente: name and tag of the image we’re building
+    `.` - specifying the build context as the current directory, and this where build expects to find the Dockerfile and the local files that the Dockerfile needs to access, in this case your python application.
```

Comment:
Bad rendering on the website:

![image](https://user-images.githubusercontent.com/1951866/178791562-d0751178-6439-41b8-8bb6-aa21b0e37789.png)


Might work:

```suggestion
    * `-t` - option and specifying its required paramente: name and tag of the image we’re building
    * `.` - specifying the build context as the current directory, and this where build expects to find the Dockerfile and the local files that the Dockerfile needs to access, in this case your python application.
```